### ISSUE TITLE
Process stops halfway

### ISSUE THREAD

**Issue by Ru194** [ENCODE-DCC/atac-seq-pipeline#251]
Hi, thanks for making this pipeline. I am wondering if you could help me sort out some issues I have. 
So, I could successfully run the test JSON file. But when I ran mine, the process didn't complete. I just had 4 output directories compared to the many that I got while I ran the test. The metadata.json file showed an error saying "workflow stopped..." 

I am using Conda environment for the pipeline and am working on a HPC cluster. My fastq files are relatively big in size. I used 4 cores and 50Gb of memory and it took more than 2 days for the process to end (with the error). Do you think it could be the memory issue? 

Also, my professor thinks if this is a memory issue, it will be too expensive since we have 90 samples to analyze and is suggesting me to not use this pipeline and execute the process manually step-wise. Do you think this pipeline requires more memory than that would be used when doing the later? What would you suggest?

Thanks a lot!

**Comment by leepc12** [comment]
Is your cluster based on SLURM? Please post your conf file `~/.caper/default.conf`.

**Comment by Ru194** [comment]
I am sorry but I am new to this cluster. I am not sure if it's based on SLURM. The only thing mentioned in its Wiki is that it's managed by IBM platform. 
Could you tell me how do I find out what the cluster is based on?
Also, the conf file is empty since I couldn't figure out the platform. 
(Since it ran successfully for the test without anything I thought it would run on my sample too)

Thank you.

**Comment by leepc12** [comment]
If you don't know about your cluster's engine, then ssh to a compute node interactively and run Caper in local mode on it.
Init caper with local mode first
```
$ caper init local
```

Please read Caper's README carefully. Follow instruction how to complete conf file `~/.caper/default.conf` there.
https://github.com/ENCODE-DCC/caper

**Comment by Ru194** [comment]
Thanks for your quick response. 
 I tried doing what you said.  However, the interactive session cannot run for more than 8 hours.I got "call-align " , "call-align_mito" and " call-read_genome_tsv" as outputs (all done without errors) and the process apparently stopped after that since that took 8 hours.

1. Would running this on a non-interactive session work (since this apparently could take around 2 days to complete, according to somebody who ran this earlier?)
2. Or could I use the outputs from just these 2 directories and analyze them further manually?

Thank you so much for your help!
